models/wiki.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	reservedWikiNames = []string{"_pages", "_new", "_edit"}
+	reservedWikiNames = []string{"_pages", "_new", "_edit", "raw"}
```

Comment:
Heya, why do we need to add `_raw` here?

Reply:
oh dear... it's not so much that raw but raw/*... hmm...

Reply:
It's sort of a shame to prevent calling something `raw` though. We've already banned `_edit` so `/raw/_edit` is unambiguous to mean edit. The issue is that delete is also not reserved - perhaps that needs to be `_delete` and then we could allow `raw`